v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/options/SourceDbToSpannerOptions.java

```diff
@@ -117,8 +117,20 @@ public interface SourceDbToSpannerOptions extends CommonTemplateOptions {
 
   void setNumPartitions(Integer value);
 
-  @TemplateParameter.Text(
+  @TemplateParameter.Integer(
       order = 9,
+      optional = true,
+      description = "The number of rows to fetch per page read for JDBC source.",
+      helpText =
+          "The number of rows to fetch per page read for JDBC source. If not set, the default of JdbcIO of 50_000 rows gets used."
```

Comment:
I don't think having an additional flag for mysql makes sense. Can we just check if this is passed and use this value ?

Reply:
The flag `fetcSize` is meant for all of the jdbc sources - mySQl, PG, oracle etc.
We have not added any flag for setting the cursor mode, as users can set it via connection properties (fetchsize needs to be set while preparing the statement and hence not a connection property) either in shard config or URL, so for that part we don't have any flag, it's just a recommendation.